include/bbcode.php

```diff
@@ -1164,7 +1164,7 @@ function ($match) use ($simplehtml){
 	$Text = preg_replace('/\<([^>]*?)(src|href)=(.*?)\&amp\;(.*?)\>/ism', '<$1$2=$3&$4>', $Text);
 
 	// sanitizes src attributes (only relative redir URIs or http URLs)
```

Comment:
Can you please update the explanatory comment above to include the cid protocol and why?